Refonte du site

Codealuxz opened the pull request:
J'ai refait tout le site pour une meilleur esthétique, une meilleur expérience et d'interactivité avec la liste des joueurs connectées

Codealuxz committed:
Refonte totale du site

Codealuxz committed:
Sauvegarde des modifications locales avant la fusion

Codealuxz committed:
Merge branch 'main' of https://github.com/Codealuxz/Website

Codealuxz committed:
Readme et tout

Codealuxz committed:
Ajout de nouvelles dépendances dans package.json et package-lock.json, y compris Radix UI, Lucide React, Motion, et Tailwind Merge. Amélioration de la gestion du menu burger dans ClientScripts.tsx avec des événements optimisés et suppression de la logique de réinitialisation. Nettoyage du code pour une meilleure lisibilité.

AxenoDev commented:
un github page ?

Margouta commented:
@AxenoDev https://openmc.codealuxz.fr/

Codealuxz commented:
> un github page ?

https://website-ixkd.onrender.com/

juste dans galerie vue que mon render et free il charge pas toutes les images vue qu'il a que  512MB de ram

AxenoDev commented:
pourquoi pas faire un simple github workflow ?

AxenoDev commented:
car la le site bug, et n'est pas fluid

Codealuxz commented:
> car la le site bug, et n'est pas fluid

C'est normal c'est render c'est nul en gratuit et comment je fait avec github

Codealuxz committed:
Mise à jour de la configuration Next.js pour le déploiement sur GitHub Pages, ajout de la gestion du chemin de base. Modification des liens dans les pages principales, de rejoindre et de captures d'écran pour utiliser le chemin de base dynamique. Amélioration de l'optimisation des images.

Codealuxz committed:
maj

Codealuxz committed:
Ajout de la gestion de basePath pour les liens dans les pages Home, Join et Screenshots, afin d'assurer une navigation correcte selon l'environnement. Amélioration de la structure du code avec l'importation de hooks de navigation.

Codealuxz committed:
rectification

Codealuxz committed:
Rectification

Codealuxz committed:
essai de re"ctification avec github workflow

Codealuxz committed:
teste 36 000

Codealuxz committed:
Merge branch 'main' of https://github.com/Codealuxz/Website

Codealuxz committed:
encore un rteste pour rectifier

Codealuxz committed:
Correction des chemins d'accès aux images dans les fichiers layout.tsx et page.tsx pour assurer une navigation correcte. Utilisation de chemins absolus pour les logos et les images.

Codealuxz committed:
Correction des chemins d'accès aux images dans layout.tsx et page.tsx pour utiliser des chemins relatifs, améliorant ainsi la cohérence des ressources graphiques.

Codealuxz committed:
Correction des chemins d'accès aux images dans plusieurs fichiers pour utiliser des chemins relatifs, assurant ainsi une meilleure cohérence et accessibilité des ressources graphiques.

Codealuxz committed:
Correction du chemin d'accès à l'image dans globals.css pour utiliser un chemin relatif, améliorant ainsi la cohérence des ressources graphiques.

Codealuxz committed:
Correction du chemin d'accès à l'image dans globals.css pour utiliser un chemin absolu, améliorant ainsi la cohérence des ressources graphiques.

Codealuxz committed:
test

Codealuxz committed:
OK TOUS LES BUGS SONT REGLER NORMALEMENT

Codealuxz committed:
Pour le pop (il s'affiche pas)

Codealuxz committed:
Update icon

Codealuxz committed:
Suppression de l'affichage des contributions dans l'outil de statistiques sur la page d'accueil.

Codealuxz committed:
Ajout d'un gestionnaire d'événements 'touchstart' pour le menu burger et les liens de navigation, améliorant ainsi l'interaction tactile sur les appareils mobiles.

Codealuxz committed:
Amélioration de la gestion des événements 'touchstart' pour le menu burger et les liens de navigation, afin d'éviter le double déclenchement sur mobile.

Codealuxz committed:
Refactorisation de la gestion des événements pour le menu burger et les liens de navigation en remplaçant 'click' et 'touchstart' par 'pointerdown', améliorant ainsi l'interaction sur tous les appareils.

Codealuxz committed:
Ajout d'une alerte de débogage lors du clic sur le menu burger pour faciliter le diagnostic des interactions utilisateur.

Codealuxz committed:
Ajout d'un état pour gérer l'ouverture du menu burger et mise à jour de la classe CSS en conséquence. Suppression de l'ancienne gestion des événements du menu burger dans ClientScripts.tsx.

Codealuxz committed:
Refactorisation de la gestion de l'état du menu burger dans plusieurs pages, ajout de la fermeture du menu lors de la navigation et amélioration de l'interaction utilisateur.

Codealuxz committed:
Ajout d'un bouton "Rejoindre le serveur" sur plusieurs pages pour améliorer l'accès à la communauté.

Codealuxz committed:
Suppression de la gestion des transitions de page dans ClientScripts.tsx pour simplifier le code et améliorer la performance.

Codealuxz committed:
Ajoute d'un SEO et 2-3 modif de style

Codealuxz committed:
Mise à jour de la description Open Graph

Codealuxz committed:
Mise à jour des métadonnées Open Graph et Twitter avec un nouveau lien d'image, ajout d'une nouvelle image binaire dans le répertoire public.

Codealuxz committed:
Renomage

Codealuxz committed:
Mise à jour des liens d'image dans les métadonnées Open Graph et Twitter pour refléter le nouveau domaine du site.

Codealuxz committed:
Ajout de styles CSS pour masquer le contenu principal et afficher un bouton de contribution. Amélioration de l'interaction utilisateur avec des ajustements de visibilité.

Codealuxz committed:
Refactorisation des styles CSS pour le bouton "Rejoindre le serveur" en ajoutant la classe burger-menu-btn sur plusieurs pages. Suppression de l'affichage par défaut du bouton et ajustement de la visibilité dans le menu burger.

EstePrime commented:
## Ce qui fonctionne bien

- La section "Contributeurs" est vraiment cool.
- La galerie est sympa.
- La page Join rend mieux & est plus complète.

## Ce qui pose problème (pour moi)

- **Transitions** :
  - Certaines transitions, notamment à l’arrivée sur la page sont pas très fluides _(le bounce est horrible)_.
  - La transition sur le changement de thème est inutile et ralentit l'user.

- **Typographie & Hiérarchie** :
  - La police "Inter", utilisée à la place de "Geist", rend moins bien globalement.
  - Sur la homepage, le contraste entre le titre et le sous-titre est insuffisant (même couleur blanche, avant gris).
  - L’espacement entre le titre et le sous-titre est TROP grand.

- **Composants** :
  - Le bouton "Contribuer" a perdu l’effet visuel présent dans la PR #2.
  - Les boutons en général sont trop gros.

- **Footer (mode clair)** :
  - Le texte manque de contraste, ca nuit à la lisibilité.
     - Le background aussi.

- **Effet de suivi de souris** :
  - Un doute sur sa pertinence, ça surcharge un peu.

Globalement, c'est sympa sinon.

# Au niveau du code
- **Évite `dangerouslySetInnerHTML`** pour injecter du JS dans `layout.tsx`. Ce n’est ni nécessaire ni sécurisé.

- **Trop de CSS inutile dans `globals.css`**, alors que **Tailwind est présent**. Privilégie les classes utilitaires pour tout ce qui est :
  - `font-size`
  - `padding`, `margin`
  - `text-align`, etc...

   Exemple: 
```css
button:hover {
  background-color: var(--accent-color-hover); 
  scale: 1.05;                           /* hover:scale-105 */
  transform: rotate(0deg);                /* hover:rotate-0 */
  box-shadow: 0 5px 25px rgba(244, 119, 150, 0.3); /* hover:shadow-pink-200/30  */
}

button svg {
  width: 16px;                            /* w-4 */
  height: 16px;                           /* h-4 */
  flex-shrink: 0;                         /* shrink-0 */
}

main {  /* tu met sur la div que tu cible*/
  margin-top: 90px;                        /* mt-[90px] */
  padding-top: 40px;                       /* pt-10 */
}

.main-container { /* tu met sur la div que tu cible*/
  display: flex;                         /* flex */
  justify-content: center;               /* justify-center */
  width: 100%;                           /* w-full */
  flex-direction: column;                /* flex-col */
  gap: 70px;                             /* gap-[70px] */
}
   ```

- **Les commentaires évidents ne servent à rien**. Exemple :

  ```ts
  // Si l'UUID n'est pas disponible, utiliser l'UUID de Steve comme fallback
  if (!player.uuid || player.uuid === 'undefined') {
    ```
   C’est évident, pas besoin de l’expliquer. (le if est deja explicite)
    
 ## la page /screenshots
Le useEffect utilisé pour injecter des données statiques est totalement inutile :
  ```ts useEffect(() => {
        // Utiliser directement les données hardcodées
        setScreenshots([
            { id: 1, filename: 'screen1.png', alt: 'Screenshot 1', width: 800, height: 600 },
            { id: 2, filename: 'screen2.png', alt: 'Screenshot 2', width: 800, height: 600 },
            { id: 3, filename: 'screen3.png', alt: 'Screenshot 3', width: 800, height: 600 },
            { id: 4, filename: 'screen4.png', alt: 'Screenshot 4', width: 800, height: 600 },
            { id: 5, filename: 'screen5.png', alt: 'Screenshot 5', width: 800, height: 600 },
        ])
    }, [])    
```

J'ai seulement regardé rapidement le code et relevé ce qui me semblait le plus bizarre.

+ Les images sont trop grosses & en web, privilégie le webp.
![image](https://github.com/user-attachments/assets/e963a16d-3365-4e21-8f34-fbc0504cd619)

AxenoDev commented on app/globals.css:
On à tailwindcss, donc pourquoi chager le globals.css comme ça ?

Codealuxz replied:
le site est de base en css du statique

AxenoDev commented on package.json:
```diff
@@ -1,36 +1,33 @@
-    "react-dom": "^19.0.0",
-    "tailwind-merge": "^3.2.0",
-    "tw-animate-css": "^1.2.5"
+    "react": "^18",
```
ta downgrade react ?
pourquoi ?

EstePrime replied:
Je pense que le fichier package.json a été copié depuis un autre projet, car Turbopack a été supprimé, certains éléments de @radix-ui sont à jour alors que d'autres ne le sont pas. & Next.js est revenu à la version 14, qui n’est pas à jour et est vulnérable (https://www.ncsc.gov.uk/news/vulnerability-affecting-nextjs-web-development-framework).

Codealuxz replied:
Oe j'avais pas fait attention

AxenoDev commented on app/page.tsx:
```diff
@@ -1,8 +1,339 @@
+
+    return (
+        <>
+            <header>
```
Fait des components pour la navbar / le footer / autres truc utiles

AxenoDev commented:
Après autre petite choses:
- Je trouve la navbar un peu trop grosse
- Dans la navbar quand c'est surligné, et que après on hover dessus, sa perd en esthétisme
![image](https://github.com/user-attachments/assets/c31318cf-c87e-41cb-acaf-0d0655b5d1e0)
- Sur la version téléphone, la navbar touche pas le haut de l'écran
![image](https://github.com/user-attachments/assets/160cb85d-31af-43fd-a5dd-6ff298b1a3f8)
- Lors de l'hover d'une image, pendant 1seconde, la border et de 1px, et juste après elle est de 2px
![image](https://github.com/user-attachments/assets/5c2e1db4-1c4e-4ada-8808-346157dae352)
- Sur la page changelog le boutons "Rejoindre le serveur" est présent deux fois
![image](https://github.com/user-attachments/assets/96d30e48-af6b-459c-9384-6dc2ba96a8c3)
- Sur le modal dans changelog, pourquoi pas mettre un padding, et mettre une sorte de "navbar" en sticky, avec le btns de close
![image](https://github.com/user-attachments/assets/11442565-21ec-4bc5-b5a8-9c41882393b4)
- Sur la div de commits dans changelogs, il n'y a pas d'animation lors du hover
![image](https://github.com/user-attachments/assets/848c5f3f-b3b2-4cfd-8d4c-ed2bb3f2b05a)
- Sur la navbar, quand elle est sur le haut de la page, ne pas mettre d'arrondi, sa renderai peut-être mieux[image](https://github.com/user-attachments/assets/61b4379c-41e4-427e-9bea-7db1ec629f34)
- la div pour le responsive de la navbar ne touche pas de bas de la page, et quand on scroll, elle bouge
![image](https://github.com/user-attachments/assets/7788f234-5f25-4faa-a68b-8d17c60e513b)
- Peut-être réduire un peu le btns 
![image](https://github.com/user-attachments/assets/9f8ed18f-fa1f-4a1c-a8c3-d7da96553742)
- sur la page galerie, le responsive, au lieu de laisser les images comme sa plus les mettres en flex-direction column
![image](https://github.com/user-attachments/assets/0b06a56f-a4bf-4283-bdde-cb3661a60c2d)
- Le responsive est mal fait sur certaine version, comme tablette
![image](https://github.com/user-attachments/assets/69685b46-f2f3-47ac-8d31-9b9809620264)

Codealuxz committed:
Modification de la Font

Codealuxz committed:
Rectif

Codealuxz committed:
rectif 2

Codealuxz committed:
Packet-lock

Codealuxz committed:
package-lock remis

Codealuxz committed:
Mise à jour des dépendances dans package.json et package-lock.json : amélioration des versions de @fontsource/geist, lucide-react, motion, ainsi que des types pour node, react et react-dom. Ajout de nouvelles versions pour les modules @next/swc et mise à jour de next à 14.2.30.

Codealuxz committed:
Enlèvement du double bouton sur le changelog

Codealuxz committed:
Modif de la description en gris et plus proche du titre, et modif du bouton contribue pour qu'il soit plus visible

Codealuxz committed:
modification de la couleur de la description (oubli de la variable de couleur)
Et ajoute de plus de passing sur les boutons de la nav bar au hover
Nav bar :
- réduction de la hauteur
- rectification sur le (elle n'était pas collée et haute)

Codealuxz committed:
Oublie dans le dernier comit

EstePrime commented on components/navigation/navbar.tsx:
```diff
@@ -68,7 +67,7 @@ export default function Navbar() {
                 </div>
                 <div className="hidden md:flex">
-                    <Button className="group" onClick={() => router.push("/join")}>
+                    <Button className="group" onClick={() => router.push("join")}>
```
Pourquoi tu utilise le router pour faire la navigation, utilise simplement https://nextjs.org/docs/pages/api-reference/components/link.

```tsx
import Link from 'next/link'
 
export default function Home() {
  return <Link href="/dashboard">Dashboard</Link>
}
```

EstePrime commented on app/components/ClientScripts.tsx:
Pour la gestion du thème, il y a déjà `next-themes` d'installé, qui fait parfaitement le travail automatiquement. Pourquoi créer son propre système ici ?

EstePrime commented on app/globals.css:
Tailwind est déjà présent. Pourquoi utiliser des classes CSS en plus de Tailwind ? Et pourquoi avoir supprimé les variables de shadcn ? Elles sont justement faites pour gérer les thèmes et faciliter l'utilisation de composants headless via shadcn

EstePrime commented on app/layout.tsx:
```diff
@@ -1,52 +1,142 @@
+                <meta name="twitter:description" content="OpenMC est un projet communautaire open-source dédié à la création d'un serveur Minecraft innovant et collaboratif." />
+                <meta name="twitter:image" content="https://openmc.fr/images/ui/logo.png" />
+                <meta name="keywords" content="Minecraft, Aywen, serveur Minecraft, open-source, serveur multijoueur, communauté Minecraft, plugins Minecraft, serveur innovant, serveur collaboratif, France, OpenMC, serveur français, hébergement Minecraft, développement Minecraft, projet communautaire, serveur gratuit, serveur Minecraft français, serveur Minecraft open-source, serveur Minecraft mu ...
+                <script
```
Désolé, mais je ne comprends pas du tout l’utilité de cette partie du code.
Je ne connais pas `cz-shortcut-listen`.

Si c’est en lien avec des erreurs d’hydratation (https://github.com/vercel/next.js/discussions/72035), il vaut mieux utiliser directement suppressHydrationWarning, qui était déjà présent auparavant.

Codealuxz replied:
C'est pour le SEO cette partie

EstePrime replied:
J'ai fail quand j'ai selectionner le code, je parle de la balise en dessous avec "cz-shortcut-listen".

EstePrime replied:
Je parlais pas des balises meta

EstePrime commented on next.config.mjs:
Pourquoi avoir supprimé `next.config.ts` pour le remplacer par un .mjs ?
Le projet est en TypeScript, donc utiliser next.config.ts est plus cohérent.

EstePrime reviewed (changes requested):
Plusieurs incohérences sont présentes dans le code & mauvaise utilisation de certains packages.

AxenoDev commented:
Pourquoi lors du hover de plusieurs éléments du site, les éléments sont grossis ?

AxenoDev commented:
avant de me demander une review, fait toutes les modifs demandés

AxenoDev commented:
![image](https://github.com/user-attachments/assets/b98bba13-e0f3-4f38-9f44-f974e4024f66)
Sa ne sert à rien d'avoir deux choses sur la navbar sinon, sa fait de trop
![image](https://github.com/user-attachments/assets/b6ca305f-e356-4bbb-bd2e-3b82f7f4eee5)
reprend celui de base avec son animation

Codealuxz commented:
> ![image](https://private-user-images.githubusercontent.com/93048526/459268343-b98bba13-e0f3-4f38-9f44-f974e4024f66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA5NjQ5NTYsIm5iZiI6MTc1MDk2NDY1NiwicGF0aCI6Ii85MzA0ODUyNi80NTkyNjgzNDMtYjk4YmJhMTMtZTBmMy00ZjM4LTlmNDQtZjk3NGU0MDI0ZjY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjI2VDE5MDQxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2OWMyMjZjZWQyZTAwMTY0Y2FmNTA0MmU4NjMyZjk2OWNhY2M3YTVhNmZjM2I3NDM4M2IwYzhkMGE5ZjhiNDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ucbvsAyAoUCYZNiqXNvi1yx_A381ycO9eKZPFG5--KI) Sa ne sert à rien d'avoir deux choses sur la navbar sinon, sa fait de trop ![image](https://private-user-images.githubusercontent.com/93048526/459268512-b6ca305f-e356-4bbb-bd2e-3b82f7f4eee5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA5NjQ5NTYsIm5iZiI6MTc1MDk2NDY1NiwicGF0aCI6Ii85MzA0ODUyNi80NTkyNjg1MTItYjZjYTMwNWYtZTM1Ni00YmJiLWJkMmUtM2I4MmY3ZjRlZWU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjI2VDE5MDQxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY2MDUyZmQwYjc2MmNiOTE0ZmJmYTkyMWI1MTBkZmNkNDYwYjVmYTkxODUwZjdmNjczNzI2MzMwZGY0YjI5MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hq5PEeVZ4uTutP3jvs8I4rwDKdbJLoI_rC_D7FLydA0) reprend celui de base avec son animation

genre j'enlève l'underline

AxenoDev commented:
> > ![image](https://private-user-images.githubusercontent.com/93048526/459268343-b98bba13-e0f3-4f38-9f44-f974e4024f66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA5NjQ5NTYsIm5iZiI6MTc1MDk2NDY1NiwicGF0aCI6Ii85MzA0ODUyNi80NTkyNjgzNDMtYjk4YmJhMTMtZTBmMy00ZjM4LTlmNDQtZjk3NGU0MDI0ZjY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjI2VDE5MDQxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2OWMyMjZjZWQyZTAwMTY0Y2FmNTA0MmU4NjMyZjk2OWNhY2M3YTVhNmZjM2I3NDM4M2IwYzhkMGE5ZjhiNDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ucbvsAyAoUCYZNiqXNvi1yx_A381ycO9eKZPFG5--KI) Sa ne sert à rien d'avoir deux choses sur la navbar sinon, sa fait de trop ![image](https://private-user-images.githubusercontent.com/93048526/459268512-b6ca305f-e356-4bbb-bd2e-3b82f7f4eee5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA5NjQ5NTYsIm5iZiI6MTc1MDk2NDY1NiwicGF0aCI6Ii85MzA0ODUyNi80NTkyNjg1MTItYjZjYTMwNWYtZTM1Ni00YmJiLWJkMmUtM2I4MmY3ZjRlZWU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjI2VDE5MDQxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY2MDUyZmQwYjc2MmNiOTE0ZmJmYTkyMWI1MTBkZmNkNDYwYjVmYTkxODUwZjdmNjczNzI2MzMwZGY0YjI5MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hq5PEeVZ4uTutP3jvs8I4rwDKdbJLoI_rC_D7FLydA0) reprend celui de base avec son animation
> 
> genre j'enlève l'underline

refait comme c'était avant, regarde sur https://openmc.fr/

Codealuxz committed:
Chosses corriger depuis le début :

Footer Le texte manque de contraste, ca nuit à la lisibilité
Dans la navbar quand c'est surligné, et que après on hover dessus, sa perd en esthétisme
Sur la navbar, quand elle est sur le haut de la page, ne pas mettre d'arrondi, sa renderai peut-être mieux
Lors de l'hover d'une image, pendant 1seconde, la border et de 1px, et juste après elle est de 2px*
Les images sont trop grosses & en web, privilégie le webp.
La police "Inter", utilisée à la place de "Geist", rend moins bien globalement.
Sur la homepage, le contraste entre le titre et le sous-titre est insuffisant (même couleur blanche, avant
gris).
L’espacement entre le titre et le sous-titre est TROP grand
Le bouton "Contribuer" a perdu l’effet visuel
Version des package downgrade
Sur la version téléphone, la navbar touche pas le haut de l'écran
Navbar un peu trop grosse
Sur la page changelog le boutons "Rejoindre le serveur" est présent deux fois
Peut-être réduire un peu le btns burger
Certaines transitions, notamment à l’arrivée sur la page sont pas très fluides (le bounce est horrible).

Codealuxz committed:
Responsive Tablette et Maj du changelog avec l'hiver et un hr pou réparer

Codealuxz commented:
C'est bon, j'ai fixé tout à peu près tout ce que vous avez demandé. Dites-moi s'il y a d'autres trucs

Codealuxz committed:
Remplacement des images PNG par des versions WebP pour améliorer la performance et la qualité visuelle. Mise à jour des styles CSS pour le composant "contributor-info" et ajout d'une transition de couleur. Suppression des fichiers d'icônes et d'images obsolètes.

Codealuxz committed:
Ajout de la propriété CSS "pointer-events: none;" à la classe .overlay pour désactiver les interactions utilisateur sur cet élément.

Codealuxz committed:
Modification de la couleur de fond principale et mise à jour des styles pour la classe .contributor. Suppression des styles obsolètes pour .contributor-info.

Codealuxz committed:
Correction du chemin d'accès de l'image du spawn du serveur OpenMC dans le composant HomePage.

Codealuxz committed:
Suppression des styles obsolètes pour le #map-viewer et mise à jour du fond de la classe body avec une nouvelle couleur et un effet de flou.

AxenoDev commented:
Par contre @Codealuxz, c'est quoi cette PR faite à 90% avec de l'IA ?
Je rappel juste que:
![image](https://github.com/user-attachments/assets/caf7b587-271b-4a4b-903d-e5bd8af8cf52)

AxenoDev commented:
De + (*car je viens de le remarquer*), quand on demande des changements, faut tous les faires, pas que faire la moitiée...
*Donc par exemple le CSS qu'il faut mettre en tailwindcss, ou le button 'Contribuer' à remettre à la normal, et pour chaque commit arreter de modifier tout le code
